masterlist.yaml

```diff
@@ -325,17 +336,16 @@ globals:
         text: 'F4SEは最新版です。'
       - lang: de
         text: 'Ihr F4SE ist aktuell.'
-  - type: error
+# Fallout 4 exe version checks
+  - <<: *fo4Version
+    subs: [ '[F4SE](http://f4se.silverlock.org)' ]
     condition: 'version("../f4se_loader.exe", "0.0.6.12", ==) and version("../Fallout4.exe", "1.10.111.0", !=)'
-    content:
-      - lang: en
-        text: 'Your version of F4SE is not compatible with your version of Fallout 4.'
-      - lang: da
-        text: 'Din version af F4SE er ikke kompatibel med din version af Fallout 4.'
-      - lang: ja
-        text: 'このバージョンのF4SEは使用中のFallout 4との互換性がありません。'
-      - lang: de
-        text: 'Ihre Version von F4SE ist nicht kompatibel mit Ihrer Version von Fallout 4.'
+  - <<: *fo4Version
+    subs: [ '[Place Everywhere](https://www.nexusmods.com/fallout4/mods/9424)' ] # v1.18.0.1114
+    condition: 'version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)'
+  - <<: *fo4Version
+    subs: [ '[Place Everywhere](https://www.nexusmods.com/fallout4/mods/9424)' ] # v1.18.0.1111
+    condition: 'version("F4SE/Plugins/place.dll", "1.18.0.1111", ==) and version("../Fallout4.exe", "1.10.111.0", !=)'
```

Comment:
The `.dll'  version is only updated when `F4SE` build is updated.

Reply:
The reason for using `<` in the version check is that it will work for any older version of `place.dll`, and not just whichever ones have been hardcoded into the masterlist. The current check will only work for `place.dll` versions 1.18.0.1111 and 1.18.0.1114, so will not flag an incompatibility if you have the newest version of Fallout 4 and `place.dll` version 1.17.1.1111. Before I introduced the unified version check, the masterlist has checks for almost a dozen different versions of `place.dll`: https://github.com/loot/fallout4/commit/8a66d3f706f3fa210ec2014862a66bb90ee3da05 – I don't want us to go back to having a check per version, please. :)

```yaml
  - <<: *fo4Version
    subs: [ '[Place Everywhere](https://www.nexusmods.com/fallout4/mods/9424)' ]
    condition: 'file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1114", <) and version("../Fallout4.exe", "1.10.114.0", ==)))'
```

Is only a single entry and covers both latest-Fallout-4-but-not-latest-`place.dll` and latest-`place.dll`-but-not-latest-Fallout-4.

```suggestion
    condition: 'file("F4SE/Plugins/place.dll") and ((version("F4SE/Plugins/place.dll", "1.18.0.1114", ==) and version("../Fallout4.exe", "1.10.114.0", !=)) or (version("F4SE/Plugins/place.dll", "1.18.0.1114", <) and version("../Fallout4.exe", "1.10.114.0", ==)))'
```

Reply:
Right I get the reasoning now. It would make it less of a maintenance issue.